src/sql/parts/dashboard/common/dashboardHelper.ts

```diff
@@ -172,4 +187,26 @@ export function filterConfigs<T extends { provider?: string | string[], edition?
 			return true;
 		}
 	});
+}
+
+/**
+ * Get registered container if it is specified as the key
+ * @param container dashboard container
+ */
+export function getOrSetContainer(container: object): object {
```

Comment:
I personally find this method name to be a bit confusing since get and set are opposites, which it seems strange to have the same function to one or the other.

Typically I see methods like getOrCreate which makes more sense to me.  Maybe there is a better name?

Reply:
I think I would change it to getDashboardContainer

---
In reply to: [170373032](https://github.com/Microsoft/sqlopsstudio/pull/766#discussion_r170373032) [](ancestors = 170373032)